runtime/vm/vmbootlib.c

```diff
@@ -159,8 +159,8 @@ sendLifecycleEventCallback(struct J9VMThread* vmThread, struct J9NativeLibrary*
 			if (0 == vmThread->javaVM->javaVM31) {
 				queryJavaVM31(vmThread->javaVM);
 			}
-			args[0]= &ffi_type_sint32;
-			args[1]= &ffi_type_sint32;
+			args[0]= &ffi_type_uint32;
+			args[1]= &ffi_type_uint32;
```

Comment:
Space before `=` (here and on lines 901-902).

Reply:
Fixed in d19d024.